plugins/spi/SPIOutput.cpp

```diff
@@ -208,12 +206,12 @@ SPIOutput::SPIOutput(const UID &uid, SPIBackendInterface *backend,
   personalities.insert(personalities.begin() + PERS_APA102_COMBINED - 1,
     Personality(APA102_SLOTS_PER_PIXEL,
       "APA102 Combined Control"));
-  personalities.insert(personalities.begin() + PERS_APA102PB_INDIVIDUAL - 1,
-    Personality(m_pixel_count * APA102PB_SLOTS_PER_PIXEL,
-      "APA102 with pixel brightness Individual Control"));
-  personalities.insert(personalities.begin() + PERS_APA102PB_COMBINED - 1,
-    Personality(m_pixel_count * APA102PB_SLOTS_PER_PIXEL,
-      "APA102 with pixel brightness Combined Control"));
+  personalities.insert(personalities.begin() + PERS_APA102_PB_INDIVIDUAL - 1,
+    Personality(m_pixel_count * APA102_PB_SLOTS_PER_PIXEL,
+      "APA102 PB Individual Control"));
+  personalities.insert(personalities.begin() + PERS_APA102_PB_COMBINED - 1,
+    Personality(m_pixel_count * APA102_PB_SLOTS_PER_PIXEL,
+      "APA102 PB Combined Control"));
```

Comment:
Perhaps "APA102 Pixel Bright Combine Ctrl", but I'm open to other suggestions.

Reply:
and 'APA102 Pixel Brightness Combined'